extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -2,23 +2,26 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-
 import * as azdata from 'azdata';
+import { EOL, homedir as os_homedir } from 'os';
+import * as path from 'path';
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
-import { DialogInfoBase, FieldType, FieldInfo, SectionInfo, LabelPosition, FontWeight, FontStyle, AzureAccountFieldInfo } from '../interfaces';
-import { Model } from './model';
-import { getDateTimeString } from '../utils';
-import { azureResource } from '../../../azurecore/src/azureResource/azure-resource';
 import * as azurecore from '../../../azurecore/src/azurecore';
+import { azureResource } from '../../../azurecore/src/azureResource/azure-resource';
+import { AzureAccountFieldInfo, AzureLocationsFieldInfo, DialogInfoBase, FieldInfo, FieldType, FontStyle, FontWeight, LabelPosition, PageInfoBase, SectionInfo } from '../interfaces';
 import * as loc from '../localizedConstants';
-import { EOL } from 'os';
+import { getDefaultKubeConfigPath, getKubeConfigClusterContexts } from '../services/kubeService';
+import { getDateTimeString, getErrorMessage } from '../utils';
+import { WizardInfoBase } from './../interfaces';
+import { Model } from './model';
 
 const localize = nls.loadMessageBundle();
 
 export type Validator = () => { valid: boolean, message: string };
 export type InputValueTransformer = (inputValue: string) => string;
-export type InputComponents = { [s: string]: { component: azdata.InputBoxComponent | azdata.DropDownComponent | azdata.CheckBoxComponent; inputValueTransformer?: InputValueTransformer } };
+export type FieldTypeComponent = azdata.InputBoxComponent | azdata.DropDownComponent | azdata.CheckBoxComponent | azdata.RadioButtonComponent;
+export type InputComponents = { [s: string]: { component: FieldTypeComponent; inputValueTransformer?: InputValueTransformer } };
```

Comment:
Why is this an object anyways? This should really be a Map or something

Reply:
Yes, but that is pre-existing, will tackle converting it to Map later.